extractor/youtube/validator.py

```diff
@@ -1,7 +1,11 @@
 import re
 
-def validate_yt(link): 
-    if(bool(re.match("https://\w{0,3}.?youtube+\.\w{2,3}/watch\?v=[\w-]{11}", link) or bool(re.match("https://youtu.be/[\w-]{11}", link))):
-            return True
-    return False
+def yt_validate(yt_link) :
+        if(bool(re.match("^https://\w{0,3}.?youtube+\.\w{2,3}/watch\?v=[\w-]{11}$", yt_link)) or bool(re.match("^https://youtu.be/", yt_link))):
+                return True
+        return False
 
+def is_short_link(yt_link):
+        if(bool(re.match("^https://youtu.be/", yt_link))):
+                return True
+        return False
```

Comment:
I'm not sure if this covers all variations. Please validate against the expressions in this link: https://stackoverflow.com/a/3726073/4396392.

Reply:
Sure I'll look into it.